### PR TITLE
feat: clear local storage data store

### PR DESCRIPTION
**Pull request by Zfinix**

### CODE REVIEW

**Comment by Zfinix** [comment]
@andrzejchm this pr is needed for pull #237

File: packages/transaction_signing_gateway/lib/storage/flutter_secure_storage_data_store.dart [hunk 1]
```diff
@@ -8,6 +8,8 @@ class FlutterSecureStorageDataStore implements SecureDataStore {
   }) : _store = storage ?? const FlutterSecureStorage();
 
   final FlutterSecureStorage _store;
+  static const iosOptions = IOSOptions(accessibility: IOSAccessibility.passcode);
```
[the remark this answers is not in the document]
Comment (Zfinix) [reply packages/transaction_signing_gateway/lib/storage/flutter_secure_storage_data_store.dart:11]: done

**Review by andrzejchm — APPROVED** [review]

**Comment by andrzejchm** [comment]
I'll merge it already, @wal33d006 feel free to add your review aswell

**Comment by wal33d006** [comment]
I had a look at it, looks good. Forgot to approve it 🙌

**Pull request merged**